app/uk/gov/hmrc/agentassurance/models/AmlsDetails.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentassurance.models
+
+import java.time.LocalDate
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.agentmtdidentifiers.model.{Arn, Utr}
+
+case class AmlsDetails(utr: Utr,
+                       supervisoryBody: String,
+                       membershipNumber: String,
+                       membershipExpiresOn: LocalDate,
+                       arn: Option[Arn]
```

Comment:
this will have two timestamps 1 mandatory on create, one optional and updated when updating details with arn